### ISSUE TITLE
Spelling error

### ISSUE THREAD

**Issue by wuehler** [blocknotary/proofofphone#5]
You have misspelled Ethereum in a few places.

**Comment by igorbarinov** [comment]
@wuehler Thank you for reporting. Now it's fixed. Any other suggestions?
Have you seen [microoracles.com](http://www.microoracles.com) ?

**Comment by wuehler** [comment]
Yes, I have, very cool vision.. I work at ConsenSys. Maybe we can chat at some point about the roadmap you have for your project.
